backend/src/main/java/ai/verta/modeldb/config/TestConfig.java

```diff
@@ -0,0 +1,11 @@
+package ai.verta.modeldb.config;
+
+public class TestConfig {
+  public DatabaseConfig database;
+
```

Comment:
Test user config is missing here
```
testUsers:
    primaryUser:
      email: 
      devKey: 
    secondaryUser:
      email: 
      devKey: 
```

Reply:
I'll add more items as I see them missing. The other ones were added because the library complains about fields in the yaml not being assigned to vars (which is good!)